docs/wiki/deployment/process-auditing.md

```diff
@@ -2,14 +2,75 @@
 
-Enabling these auditing features requires additional configuration to osquery. The `process_events` and `socket_events` tables use the same event-based architecture as the [File Integrity Monitoring (FIM)](../deployment/file-integrity-monitoring.md). To read more about how event-based tables are created and designed, check out the osquery [Table Pubsub Framework](../development/pubsub-framework.md). On all supported platforms, process events are abstracted into the  [`process_events`](https://osquery.io/schema/2.11.2#process_events) table. Similarly, socket events are abstracted into the [`socket_events`](https://osquery.io/schema/2.11.2#process_events) table.
+The `process_events` and `socket_events` tables use the same event-based architecture as the [File Integrity Monitoring (FIM)](../deployment/file-integrity-monitoring.md). To read more about how event-based tables are created and designed, check out the osquery [Table Pubsub Framework](../development/pubsub-framework.md). On all supported platforms, process events are abstracted into the  [`process_events`](https://osquery.io/schema/2.11.2#process_events) table. Similarly, socket events are abstracted into the [`socket_events`](https://osquery.io/schema/2.11.2#process_events) table.
 
 To collect process events add a query like:
 ```sql
 SELECT * FROM process_events;
 ```
 to your query schedule, or to a query pack.
 
+Enabling these auditing features requires additional configuration to
+osquery, and may have performance impact. See the OS specific sections
+for guidance.
+
+## General Troubleshooting
+
+
+Though some testing can be performed via `osqueryi`, `osqueryi` and
+`osqueryd` operate independently. Configuration used on one must be
+propagated to the other.
+
+### Examine configuration flags
+
+```
+osquery> select * from osquery_flags where name in ("disable_events", "disable_audit");
++----------------+------+---------------------------------------------------+---------------+-------+------------+
+| name           | type | description                                       | default_value | value | shell_only |
++----------------+------+---------------------------------------------------+---------------+-------+------------+
+| disable_audit  | bool | Disable receiving events from the audit subsystem | true          | false | 0          |
+| disable_events | bool | Disable osquery publish/subscribe system          | false         | false | 0          |
++----------------+------+---------------------------------------------------+---------------+-------+------------+
+```
+
+### Examine event table
+
+osquery keeps state about the events subsystem in the `osquery_events`
+table. The `subscriptions` is of note here. This example is from a
+macOS machine with events enabled. (But no events detected)
```

Comment:
It's not clear what the troubleshooting goal is here. Perhaps something additional like:

"At this point you should attempt to trigger an event. Then confirm that the event count is non-0."

Reply:
I'm trying to give people the ability to diagnose what's happening. I'll try to reword